Example/Example/Examples/RequestScalingHeader.swift

```diff
@@ -12,55 +12,80 @@ import ScalingHeaderScrollView
 struct RequestScalingHeader: View {
 
     @Environment(\.presentationMode) var presentationMode
-    
     @State private var isLoading: Bool = false
-    @State private var image: UIImage = UIImage()
-    @ObservedObject private var imageLoader = ImageLoaderService()
-    
+    @State private var colorSet: [Color] = [.red, .blue, .green, .black, .pink, .purple, .yellow, .red]
+
     var body: some View {
         ZStack(alignment: .topLeading) {
             ScalingHeaderScrollView {
                 ZStack {
                     Rectangle()
-                        .fill(Color.gray.opacity(0.15))
-                    Image(uiImage: image)
```

Comment:
why did you remove the header? the library is about the header which scales, and all of the examples need to show that - that's the main point. load more is just an extra feature. please restore the header